diamond-api/src/bin/diamond-api.rs

```diff
@@ -32,10 +32,15 @@ fn run(args: Args) -> io::Result<()> {
 
     let listen = format!("127.0.0.1:{}", &args.port);
 
+    let ctx = Context {
+        args: args.clone(),
```

Comment:
args should not be cloned here. It could be consumed by Context.

Example changes:
```
     let ctx = Context {
-        args: args.clone(),
         walker: Walker::File(args.path.clone()),
+        args,
     };
```

Reply:
Done